bodhi/server/schemas.py

```diff
@@ -213,6 +213,7 @@ class SaveUpdateSchema(CSRFProtectedSchema, colander.MappingSchema):
     notes = colander.SchemaNode(
         colander.String(),
         validator=colander.Length(min=2),
+        missing_msg='Required description for the update.'
```

Comment:
Maybe we can change this to `A description is required for the update`. I find it easier to understand as an error message